<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0" xmlns="http://www.qt.io/ns/ui">
    <class>ColorsWindow</class>
    <widget class="QWidget" name="ColorsWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>463</width>
                <height>170</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Color Palette</string>
        </property>
        
       <widget class="QWidget" name="gridLayoutWidget">
            <property name="geometry">
                <rect>
                    <x>30</x>
                    <y>30</y>
                    <width>361</width>
                    <height>100</height>
                </rect>
            </property>

            <layout class="QGridLayout" name="gridLayout">

                <!-- row 0 -->
                <item row="0" column="0">
                    <layout class="QVBoxLayout" name="verticalLayout_2">
                        <item>
                            <widget class="QLabel" name="label">
                                <property name="text">
                                    <string>Select Environment Image Map: </string>
                                </property>
                            </widget>
                        </item>
                    </layout>
                </item>
                <item row="0" column="1">
                    <widget class="QPushButton" name="select_button">
                        <property name="text">
                            <string>Select</string>
                        </property>
                    </widget>
                </item>

                <item row="1" column="0">
                    <widget class="QLabel" name="warnings">
                        <property name="text">
                            <string/>
                        </property>
                    </widget>
                </item>

                 <!-- Row 1: Color Picker Button -->
                <item row="1" column="0">
                    <widget class="QPushButton" name="color_picker_button">
                        <property name="text">
                            <string>Pick Color</string>
                        </property>
                    </widget>
                </item>

                <!-- Row 1: Color Display Box (QFrame) -->
                <item row="1" column="1">
                    <widget class="QFrame" name="color_display_frame">
                        <property name="frameShape">
                            <enum>QFrame::StyledPanel</enum>
                        </property>
                        <property name="frameShadow">
                            <enum>QFrame::Raised</enum>
                        </property>
                        <property name="styleSheet">
                            <string>background-color: #eb4034;</string>
                        </property>
                        <property name="minimumSize">
                            <size>
                                <width>50</width>
                                <height>25</height>
                            </size>
                        </property>
                    </widget>
                </item>

            </layout>
        </widget>

        <!-- Apply Button-->
        <widget class="QPushButton" name="apply_button">
            <property name="geometry">
                <rect>
                    <x>20</x>
                    <y>130</y>
                    <width>100</width>
                    <height>32</height>
                </rect>
            </property>
            <property name="text">
                <string>Apply</string>
            </property>
        </widget>

    </widget>
    <resources/>
    <connections/>
</ui>
